### ISSUE TITLE
Overlapping graphics with<Select> in <Form.Item ... hasFeedback={true}>

### ISSUE THREAD

**Issue by JesperWe** [ant-design/ant-design#4431]
Current result: 

![image](https://cloud.githubusercontent.com/assets/1643015/21567704/7c399536-ceae-11e6-8a06-448879769122.png)

I am currently fixing this with
```
.has-feedback .ant-select .ant-select-arrow {
    right: 28px;
}
```
to become

![image](https://cloud.githubusercontent.com/assets/1643015/21567730/a2d2ba38-ceae-11e6-9948-a0e2b5a43b33.png)

This might not be the preferred solution, but something like it is needed.

**Comment by afc163** [comment]
Could you give us a PR for this?

**Comment by JesperWe** [comment]
I could try... Would you prefer this to go in `components/select/style/index.less` or `components/form/style/index.less` ?

**Comment by afc163** [comment]
`components/form/style/index.less`

**Comment by sduworm** [comment]
This reappeared at v2.12.1 ！@afc163

**Comment by afc163** [comment]
Fixed in 2.12.2 again.
